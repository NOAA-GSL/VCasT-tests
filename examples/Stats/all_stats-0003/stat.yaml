# Date and Time Settings
start_date: "2024-04-01_02:00:00"  # Start date of the data processing
end_date: "2024-04-01_02:00:00"    # End date of the data processing
interval_hours: "1"                # Interval for processing (e.g., hourly, 6-hourly)

start_lead_time: 1
end_lead_time: 1
interval_lead_time: 1

var: "T2M"                    # Forecast variable name

# Forecast Configuration
fcst_file_template: "../input_files/REF/ref_{year}{month}{day}{hour}_f0{lead_time}.nc"  # Reference file template
fcst_model: "resHRRR"

# Reference Configuration
ref_file_template: "../input_files/HRRR/hrrr.{valid_year}{valid_month}{valid_day}{valid_hour}.wrfprsf00.T2M.grib2"
ref_model: "HRRR"

# Output Configuration
output_dir: "."  # Directory to save outputs
output_filename: "stat.out"  # Output file template

# Statistical Metrics
stat_type: "det"
stat_name:
  - "rmse"
  - "bias"
  - "quantiles"
  - "mae"
  - "gss:290:290:2"
  - "fbias:290:290:2"
  - "pod:290:290:2"
  - "far:290:290:2"
  - "csi:290:290:2"
  - "corr"
  - "stdev"
  - "sr:290:290:2"

threshold:

# Variable Threshold and Radius
var_threshold: 290                  # Threshold value for analysis
var_radius: 2                        # Influence radius for calculations (grid points)

# Grid and Interpolation Settings
interpolation: true               # Whether to interpolate data
target_grid: "../input_files/REF/ref_2024040102_f001.nc"   # Target grid specification

# Parallel Processing
processes: 100                     # Number of processes to run in parallel
