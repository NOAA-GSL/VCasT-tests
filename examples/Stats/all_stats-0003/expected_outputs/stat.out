date	lead_time	rmse	bias	25p	50p	75p	IQR	LW	UW	mae	gss	fbias	pod	far	csi	corr	stdev	sr
2024-04-01 02:00:00	1	5.47031151505105	1.0080553622327304	-0.8772479068352794	0.10654485788361967	3.0909271240234375	3.968175030858717	-6.829510453123355	9.043189670311513	3.643547958769518	0.0	nan	nan	1.0	0.0	0.6173284275487291	5.9784155	0.0
